.g8/checkboxPage/generated-test/models/pageviews/$className$ViewModelSpec.scala

```diff
@@ -0,0 +1,42 @@
+package models.pageviews
+
+import builders.UserAnswersBuilder.aUserAnswers
+import forms.$className$FormProvider
+import models.{$className$, NormalMode}
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.must.Matchers
+import pages.$className$Page
+
+class $className$ViewModelSpec extends AnyFreeSpec with Matchers {
+
+  private val anyMode = NormalMode
+  private val formProvider = new $className$FormProvider()
+
+  private val underTest = $className$ViewModel
+
+  ".apply(...)" - {
+    "must return ViewModel with pre-filled form when BusinessTypePage answer available" in {
```

Comment:
BusinessTypePage needs to be $className$Page (commenting here, but it's repeated in other places)

Reply:
Updated in all places